### PR TITLE
Added telefreedom implants

### PR DESCRIPTION
**Pull request by FluffyGhoster**
Added telefreedom implants, including a box with (almost) everything you need to make them work for 4 people, and added it to the uplink.
Fixed implanters not taking the implant and dropping it to the ground when hit by one (while keeping it referenced).

### CODE REVIEW

**Comment by FluffyGhoster** [comment]
!review

File: html/changelogs/fluffyghost-telefreedomimplant.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+# Please surround your changes in  double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - rscadd: "Added telefreedom implants, including a box with (almost) everything you need to make them work for 4 people, and added it to the uplink."
```
Comment (NonQueueingMatt) [inline html/changelogs/fluffyghost-telefreedomimplant.yml:58]: Probably describe what these actually do?

**Review by NonQueueingMatt — APPROVED** [review]

File: html/changelogs/fluffyghost-telefreedomimplant.yml (hunk 1 above)
Comment (FluffyGhoster) [inline html/changelogs/fluffyghost-telefreedomimplant.yml:58]:
```suggestion
  - rscadd: "Added telefreedom implants, including a box with (almost) everything you need to make them work for 4 people, and added it to the uplink. The implant allows you to teleport to the linked telepad (you link it by clicking on the telepad with the implent in hand before implanting) one time before melting."
```

Comment (FluffyGhoster) [reply html/changelogs/fluffyghost-telefreedomimplant.yml:58]: @NonQueueingMatt this works?

File: code/datums/uplink/implants.dm [hunk 2]
```diff
@@ -60,3 +60,10 @@
 	bluecrystal_cost = 5
 	desc = "A augment implanter, with a Hivenet defense augment and electronic warfare suite. ONLY WORKS ON VAURCA!"
 	path = /obj/item/device/augment_implanter/hivenet_warfare
+
+/datum/uplink_item/item/implants/telefreedom_kit
+	name = "Telefreedom Kit"
+	bluecrystal_cost = 15
+	item_limit = 1
+	desc = "A box containing a telefreedom full kit, with everything you need for 4 people. You supply the screwdriver and have to build it."
```
Comment (DreamySkrell) [inline code/datums/uplink/implants.dm:68]: this should explain what is a "telefreedom kit"

Comment (DreamySkrell) [reply code/datums/uplink/implants.dm:68]: cause someone who had never used it, would have no idea what it is and what does it do, just by looking at this text in the uplink program

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**